### ISSUE TITLE
Remove "shell parameter expansion" from exercises? 

### ISSUE THREAD

**Issue by calebhailey** [sensu/sensu-enterprise-workshop#79]
## Suggested Improvement

TL;DR: environment variables should look like `${SENSU_API_URL}` not `${SENSU_API_URL:-http://127.0.0.1:8080}`. 

Many of the `curl` examples in the workshop use [shell parameter expansion](https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html) (e.g. `"${SENSU_API_URL:-http://127.0.0.1:8080}`), which effectively provide "fallback" or "default" values in the event that the referenced environment variable is not present. Since we we began working on the workshop we've more or less standardized on the use of an `.envrc` or `.envrc.ps1` file to provide the required environment variables for all exercises. Perhaps we should use the simpler form of environment variables now? 

## Applicable Lessons

- [ ] Lesson 01 completed or in-progress
- [ ] Lesson 02 completed or in-progress
- [x] Lesson 03 completed or in-progress
- [ ] Lesson 04 completed or in-progress
- [x] Lesson 05 completed or in-progress
- [x] Lesson 06 completed or in-progress
- [x] Lesson 07 completed or in-progress
- [ ] Lesson 08 completed or in-progress
- [ ] Lesson 09 completed or in-progress
- [ ] Lesson 10 completed or in-progress
- [ ] Lesson 11 completed or in-progress
- [ ] Lesson 12 completed or in-progress
- [ ] Lesson 13 completed or in-progress
- [ ] Lesson 14 completed or in-progress
- [ ] Lesson 15 completed or in-progress
- [ ] Lesson 16 completed or in-progress
